src/me/mrCookieSlime/ExoticGarden/PlantData.java

```diff
@@ -2,22 +2,12 @@
 
 public class PlantData {
 
-	private byte data;
 	private String texture;
 
-	public PlantData(byte data) {
-		this.data = data;
-		this.texture = "NO_SKULL_SPECIFIED";
-	}
-
 	public PlantData(String texture) {
 		this.texture = texture;
 	}
 
-	public byte toByte() {
-		return data;
-	}
-
 	public String getTexture() {
 		return texture;
 	}
```

Comment:
Isn't this entire class basically useless if it only stores one String?
I think we can remove this class.

Otherwise, this PR is ready for being merged!
Happy holidays btw